challenges/2021-07-20-balanced-brackets/solutions/python/laurelin/laurelin.py

```diff
@@ -0,0 +1,30 @@
+def balanced_brackets(input: str) -> bool:
+    if len(input) == 0:
+        return True
+
+    stack = []
+    brackets = {'(', ')', '[', ']', '{', '}'}
+    bracket_pair = {')': '(', ']': '[', '}': '{'}
+
+    for char in input:
+        if char not in brackets:
+            continue
+
+        is_closed = char in bracket_pair
+        size = len(stack)
+
+        if size == 0 and is_closed:  # if there are no opening brackets, but we found a closing bracket
+            return False
+
+        if not is_closed:
+            stack.append(char)
+        else:
+            bracket = stack.pop()
+            if bracket_pair[char] != bracket:
+                return False
+
+    size = len(stack)
+    if size == 0:
+        return True
+    else:
+        return False
```

Comment:
This looks great! As a point of trivia, if you're like me and like to condense your code a bit, I believe your last line in the function could be `return True if size == 0 else return False`, which is a nice bit of syntax.